benchmark-service/benchmark/benchmark.py

```diff
@@ -0,0 +1,58 @@
+import os
+from pathlib import Path
+from typing import Any, AsyncIterator, Dict
+
+import aiohttp_jinja2
+import jinja2
+from aiohttp import web
+import logging
+
+app = web.Application()
```

Comment:
delete the app defined here. You have it defined below in your `init_app()` function.

Reply:
Sorry. I wasn't clear enough. Where `init_app()` was in the file was perfect. I just wanted you to delete the `app = web.Application()` where app is a global variable that is not used anywhere.

Reply:
ah, I did at first but it was used in .setup and it was giving an error so I wasn't sure